compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1598,6 +1603,11 @@ impl Debug for Statement<'_> {
                     CoverageKind::Unreachable => write!(fmt, "Coverage::Unreachable for {:?}", rgn),
                 }
             }
+            CopyNonOverlapping(box crate::mir::CopyNonOverlapping {
+                ref src,
+                ref dst,
+                ref size,
+            }) => write!(fmt, "src {:?} -> dst {:?}, {:?} bytes", src, dst, size),
```

Comment:
make this look like a proper function call. As this shows up in mir textual output, consistency is fairly important.